flink-table/flink-table-planner-blink/src/test/scala/org/apache/flink/table/api/batch/ExplainTest.scala

```diff
@@ -40,6 +40,12 @@ class ExplainTest(extended: Boolean) extends TableTestBase {
   val LONG = new BigIntType()
   val INT = new IntType()
 
+  @Before
+  def before(): Unit = {
+    util.tableEnv.getConfig.getConfiguration.setInteger(
+      ExecutionConfigOptions.TABLE_EXEC_RESOURCE_DEFAULT_PARALLELISM, 4)
```

Comment:
What's this used for?

Reply:
StreamPartitioner in StreamGraph produced by explaining is influenced by parallelism. So env parallelism should be config.